### PR TITLE
Adding skflow to tensorflow/contrib

### PR DESCRIPTION
**Pull request by ilblackdragon**

### CODE REVIEW

**Comment by ilblackdragon** [comment]
Hi @terrytangyuan @makseq @dansbecker @elqursh @dgboy2000 @cbonnett @ivallesp @frol @liyongsea,

We decided to move skflow into tensorflow - for better convergence of APIs and generally better support.
Can you please comment in this thread with "I signed it! message for CLA to propagate for you to tensorflow project (this will make you contributor of tensorflow and keep your stats).

Thanks!

**Comment by terrytangyuan** [comment]
I signed it!

**Comment by dansbecker** [comment]
I signed it!

**Comment by ivallesp** [comment]
I signed it!

**Comment by makseq** [comment]
I signed it!

**Comment by elqursh** [comment]
I signed it!

**Comment by dgboy2000** [comment]
I signed it!

**Comment by frol** [comment]
I signed it!

**Comment by cbonnett** [comment]
I signed it!

**Comment by ilblackdragon** [comment]
Jenkins, test this please

**Comment by ilblackdragon** [comment]
Jenkins, test this please

**Comment by ilblackdragon** [comment]
Jenkins, test this please

**Comment by ilblackdragon** [comment]
Jenkins, test this please

File: tensorflow/tools/ci_build/install/install_pip.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```
Comment (caisq) [inline tensorflow/tools/ci_build/install/install_pip.sh]: No need for this file once https://github.com/tensorflow/tensorflow/pull/1472 is in.

File: tensorflow/tools/ci_build/Dockerfile.cpu [hunk 2]
```diff
@@ -8,6 +8,7 @@ RUN /install/install_bootstrap_deb_packages.sh
 RUN add-apt-repository -y ppa:openjdk-r/ppa
 RUN /install/install_deb_packages.sh
 RUN /install/install_bazel.sh
+RUN /install/install_pip.sh
```
Comment (caisq) [inline tensorflow/tools/ci_build/Dockerfile.cpu]: No need for this line once https://github.com/tensorflow/tensorflow/pull/1472 is in.

Comment (caisq) [inline tensorflow/tools/ci_build/Dockerfile.cpu]: 1) See comment below. No need to put this in a separate file. Just do pip install here. 
2) The same change needs to be made in Dockerfile.gpu in the same directory.

**Comment by caisq** [comment]
Sorry about the change in suggestions, but PR 1472, which is already merged after discussion, ended up including all the deb and pip packages required for the builds and tests here. As such, please remove all changes to the following files: 
tensorflow/tools/ci_build/Dockerfile.cpu
tensorflow/tools/ci_build/Dockerfile.gpu
tensorflow/tools/ci_build/install/install_deb_packages.sh
tensorflow/tools/ci_build/builds/test_installation.sh
tensorflow/tools/ci_build/builds/pip.sh
Thanks.

**Comment by terrytangyuan** [comment]
@ilblackdragon How about the `"cla: no"` thing? Can we still merge it which actually makes sense since they signed `cla` from skflow already?

**Comment by martinwicke** [comment]
Everybody has signed the Google CLA when contributing to skflow.

**Pull request merged**